wlib/AttrListDlg.cpp

```diff
@@ -367,21 +370,30 @@ std::istream& safeGetline(std::istream& is, std::string& t)
 
 IModule * DoConfirmImportDlg(HWND hwndParent, const boost::filesystem::path & path)
 {
-	std::ifstream ifs(path.c_str());
-	if (ifs) {
+	CUnicodeFile file;
+	if (file.Open(path.c_str()))
+	{
+		std::_tstring content;
+		file.Read(content);
+		typedef std::vector<std::_tstring> split_vector_type;
+		split_vector_type SplitVec;
+		boost::algorithm::split(SplitVec, content, boost::algorithm::is_any_of("\r\n"), boost::algorithm::token_compress_on);
```

Comment:
It might be safer to replace all `\r\n` with `\n` and then split on `\n` (I am not sure how consistent we have been with that though)